client/js/store/modules/game.js

```diff
@@ -131,6 +131,10 @@ export default {
     hasGlassesEight(state, getters) {
       return getters.player.faceCards.filter((card) => card.rank === 8).length > 0;
     },
+    isRanked(state, getters, rootState) {
+      const game = rootState['gameList'].games.find(({ id }) => id === state.id);
+      return game ? game.isRanked : false;
+    },
```

Comment:
Now a getter 😎

Reply:
It makes more sense as state than a getter imo. Why do a linear lookup of the games and compare by id when this module already has the 'full' data for a single game (which could include it's ranked state)

Reply:
I didn't see those anywhere in the state and I wasn't sure if this feature should be hoisting those up at this point. I do agree it would be better to have it in there. Any reason why we didn't add it in the first place since it's directly related to the game object?